man/udevadm.xml

```diff
@@ -564,6 +564,18 @@
             <xi:include href="version-info.xml" xpointer="v251"/>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>-i</option></term>
+          <term><option>--include-parents</option></term>
+          <listitem>
+            <para>Trigger parent devices of filtered devices even if the parents
+            won't match the filter condition.
+            This is useful if we are interested to limit the coldplug activities to
+            some devices or subsystems.</para>
+
+            <xi:include href="version-info.xml" xpointer="v257"/>
```

Comment:
Please move this to v258. It is slightly late this to be included in v257, unfortunately.